Hotels/Controllers/HotelsController.cs

```diff
@@ -88,5 +88,29 @@ public async Task<ActionResult<Hotel>> DeleteHotel(long id)
 
             return hotel;
         }
+
+        // GET: api/Hotels/5/Rooms
+        [HttpGet("{id}/Rooms")]
+        public async Task<ActionResult<List<HotelRoom>>> GetHotelRooms(long hotelId)
+        {
+            var hotel = await repository.GetHotelRoomsById(hotelId);
+            return hotel.HotelRooms;
+        }
+
+        // POST: api/Hotels/5/Rooms/12
+        [HttpPost("{hotelId}/Rooms/{roomId}")]
+        public async Task<ActionResult> AddRoom(long hotelId, long roomId)
+        {
+            await repository.AddRoomAsync(hotelId, roomId);
```

Comment:
Too easy to confuse `roomId` with `roomNumber`
```suggestion
        [HttpPost("{hotelId}/Rooms")]
        public async Task<ActionResult> AddRoom(long hotelId, CreateHotelRoom hotelRoom)
        {
            await repository.AddRoomAsync(hotelId, hotelRoom);
```

`CreateHotelRoom` would look something like this:

https://github.com/DeltaVCode/cr-dotnet-401d2/blob/b88bf612a03e135f94dd7e08f687e5ec093fbbdd/Class14/Demo/Web/Models/CreateGrade.cs#L5-L12